docs/pontoon.md

```diff
@@ -0,0 +1,79 @@
+# wagtail-localize-pontoon
```

Comment:
This will be rewritten, merging it in so it could be used as a reference for that rewrite.